app/views/map/index.html.haml

```diff
@@ -114,6 +114,7 @@
   .center
     Pro zobrazení událostí musíte přiblížit mapu.
 
+= javascript_include_tag 'categories_legend.js'

```

Comment:
I would prefer to put it to assets (application.js) event it is used only in map.